docker/set-role-Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM curlimages/curl:latest
```

Comment:
is this used?

Reply:
It's related to set-role container and will be used.

Reply:
Now it's used.